src/plugins/abrt-action-install-debuginfo-to-abrt-cache.c

```diff
@@ -75,10 +75,9 @@ int main(int argc, char **argv)
     };
     const unsigned opts = libreport_parse_opts(argc, argv, program_options, program_usage_string);
 
+    const uid_t euid = geteuid();
```

Comment:
Is there a reason to move this line?

Reply:
Nope. I moved it, because I removed it, but later I found that it is actually needed.